src/rpdk/core/contract/resource_client.py

```diff
@@ -277,8 +282,18 @@ def assert_failed(status, response):
         return error_code
 
     @staticmethod
-    def make_request(desired_resource_state, previous_resource_state, **kwargs):
+    def make_request(
+        desired_resource_state,
+        previous_resource_state,
+        region,
+        partition,
+        account_id,
+        **kwargs
+    ):
         return {
+            "region": region,
+            "awsPartition": partition,
+            "awsAccountId": account_id,
             "desiredResourceState": desired_resource_state,
             "previousResourceState": previous_resource_state,
```

Comment:
I'd prefer to preserve the order of the inputs

Reply:
I don't think order matters here. But, I can go ahead and change it.

Reply:
There might be some expectation in the code (I hope there's *not*) that uses positional values from the map instead of key lookups.